gradle-enterprise-conventions-maven-extension/extension/src/main/java/com/example/CiUtils.java

```diff
@@ -0,0 +1,22 @@
+package com.example;
+
+import java.util.Optional;
+
+final class CiUtils {
+
+    private CiUtils() {
+    }
+
+    static boolean isCi() {
```

Comment:
We already implement this (and more thoroughly) in the Common Custom User Data Maven Extension: https://github.com/gradle/common-custom-user-data-maven-extension/blob/main/src/main/java/com/gradle/CiUtils.java#L8

If we exposed that, we could use it here and ditch this whole class

Reply:
+1 to reusing the CCUD functionality. Note that there's an additional implementation of `isCI` in the GE config for this actual project. So in essence we are asking the developer that is copying this repo to implement `isCI` twice even though it's already fully implemented in the CCUD plugin.

Reply:
I made this PR over in Common Custom User Data Maven Extension to do this: https://github.com/gradle/common-custom-user-data-maven-extension/pull/111

Reply:
The PR was closed.